deployment/src/main/java/dev/ebullient/micrometer/deployment/binder/mpmetrics/MetricDotNames.java

```diff
@@ -0,0 +1,98 @@
+package dev.ebullient.micrometer.deployment.binder.mpmetrics;
+
+import java.util.*;
+
+import org.jboss.jandex.AnnotationInstance;
+import org.jboss.jandex.ClassInfo;
+import org.jboss.jandex.DotName;
+
+import io.quarkus.arc.processor.BuiltinScope;
+
+/**
+ * The microprofile API must remain optional.
+ *
+ * Avoid importing classes that import MP Metrics API classes.
+ */
+public class MetricDotNames {
```

Comment:
I think the current metrics extension has a class like this too, could be interesting to see if there's a way to reuse, but not a huge deal

Reply:
both are simple and full of strings that matter to one or the other. I think repetition is fine, in this case.

Reply:
Yep, nbd